### PR TITLE
fix possession in cluster-deployment instruction

### PR DESCRIPTION
**Pull request by hanqiuzh**
Signed-off-by: Hanqiu Zhang <hanzhang@redhat.com>

### CODE REVIEW

**Comment by mikeshng** [comment]
@hanqiuzh can you also update the weight of install cluster lifecycle so it's below the install klusterlet. Thanks.
Currently, its sitting ontop of Install cluster manager.

File: content/getting-started/install-cluster-lifecycle.md [hunk 1]
```diff
@@ -70,9 +70,9 @@ You must meet the following prerequisites to use the cluster lifecycle:
    spec:
      hubAcceptsClient: true
    ```
-   Replace the `CLUSTER_NAME` with the cluster's name you are using.
+   Replace the `CLUSTER_NAME` with the cluster name that you are using.
 
-   **Note**: `CLUSTER_NAME` should be same as both the clusterdeployment's resource name and the clusterdeployment's namespace name.
+   **Note**: `CLUSTER_NAME` should be same as both the `ClusterDeployment` resource name and the `ClusterDeployment` namespace name.
```
Comment (dockerymick) [inline content/getting-started/install-cluster-lifecycle.md:75]: Thanks for recreating a PR Han :) May you also replace the period (.) with a colon (:) for step 3 in the _Install managedcluster-import-controller from the source files_ section?

**Review by dockerymick — COMMENTED** [review]
Thanks for the changes!!

File: content/getting-started/install-cluster-lifecycle.md (hunk 1 above)
Comment (hanqiuzh) [reply content/getting-started/install-cluster-lifecycle.md:75]: yes, updated.

**Review by dockerymick — APPROVED** [review]
looks good!

**Pull request merged**